src/components/PostDetails/PostDetails.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react';
+import { NewCommentForm } from '../NewCommentForm';
+import './PostDetails.scss';
+import { Post, Comment } from '../../types';
+import { getComments, delComment } from '../../api/comments';
+import { getPostbyId } from '../../api/posts';
+
+type Props = {
+  postId: number | undefined;
+};
+
+export const PostDetails: React.FC<Props> = ({
```

Comment:
It would be a good idea to review the code’s indentation and format it to improve readability and maintainability. This makes it easier to understand the code flow and helps in future code reviews.